vanilla_installer/windows/main_window.py

```diff
@@ -154,15 +158,27 @@ def next(self, widget=None, fn=None, *args):
 
         page = self.carousel.get_nth_page(cur_index + 1)
         self.carousel.scroll_to(page, True)
+        try:
+            page.gen_deltas()
+        except:
+            pass
 
     def back(self, *args):
         logger.info("Going to previous page")
 
         cur_index = self.carousel.get_position()
+        try:
+            self.carousel.get_nth_page(cur_index).del_deltas()
+        except:
+            pass
```

Comment:
I am fine with not handling an exception but should be at least logged

Reply:
It's not error catching, it's to catch when any default doesn't implement `del_deltas()` (Like the welcome page)

Reply:
fine then

Reply:
I think it could still be better than ignoring every error.
Either a check if the method exists or just an empty method in the classes that don't support it.